gemrb/GUIScripts/LevelUp.py

```diff
@@ -245,6 +245,10 @@ def OpenLevelUpWindow():
 			Specialist = 0
 			if CommonTables.KitList.GetValue (Kit, 7) == 1: # see if we're a kitted mage
 				Specialist = 1
+
+			if Spellbook.HasSorcererBook (pc, Classes[0]):
```

Comment:
This should be `Classes[i]` and I should not be that lazy (the window opening below needs to change to loop over Classes). Just in case someone ever makes a multiclass mod that doesn't have sorcerers at the first slot.